flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+          x-field-uid: 4
+    Flow.RSVP.PathObjectsClassRsvpHop:
+      description: >-
+          C-Type is specific to a class num.
+      type: object 
+      properties:
+        c_type: 
+          $ref: '#/components/schemas/Flow.RSVP.PathObjectsRsvpHopCType'
+          x-field-uid: 1 
+    Flow.RSVP.PathObjectsRsvpHopCType: 
+      description: >-
+        Object for RSVP_HOP class.
+        Currently supported c-type is IPv4 (1).
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          default: ipv4
+          x-enum:
+            ipv4:
+              x-field-uid: 1
+        ipv4:
+          $ref: '#/components/schemas/Flow.RSVP.PathRsvpHopIpv4'
+          x-field-uid: 2
+    Flow.RSVP.PathRsvpHopIpv4:
+      description: >-
+        IPv4 RSVP_HOP object: Class = 3, C-Type = 1
+      type: object
+      properties:
+        ipv4_next_previous_hop_address:
+          x-field-pattern:
+            description: >-
+              The IPv4 address of the interface through which the last RSVP-knowledgeable hop forwarded this message.
+            format: ipv4
+            default: 0.0.0.0
+            features: [count]
+          x-field-uid: 1
+        logical_interface_handle:
+          x-field-pattern:
+            description: >-
+              Logical Interface Handle (LIH) is used to distinguish logical outgoing interfaces.
+              A node receiving an LIH in a Path message saves its value and returns it in the HOP objects of subsequent Resv messages sent to the node that originated the LIH.
+              The LIH should be identically zero if there is no logical interface handle.
+            format: integer
+            default: 0
+            length: 32
+            features: [count]
+          x-field-uid: 2
+    Flow.RSVP.PathObjectsClassTimeValues:
+      description: >-
+          C-Type is specific to a class num.
+      type: object 
+      properties:
+        c_type: 
+          $ref: '#/components/schemas/Flow.RSVP.PathObjectsTimeValuesCType'
+          x-field-uid: 1 
+    Flow.RSVP.PathObjectsTimeValuesCType:
+      description: >-
+        Object for TIME_VALUES class.
+        Currently supported c-type is Type 1 Time Value (1).
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          default: type_1_time_value
+          x-enum:
+            type_1_time_value:
+              x-field-uid: 1
+        type_1_time_value:
+          $ref: '#/components/schemas/Flow.RSVP.PathTimeValuesType1'
+          x-field-uid: 2
+    Flow.RSVP.PathTimeValuesType1:
+      description: >-
+        TIME_VALUES Object: Class = 5, C-Type = 1
+      type: object
+      properties:
+        refresh_period_r:
+          x-field-pattern:
+            description: >-
+              The refresh timeout period R used to generate this message;in milliseconds.
+            format: integer
+            default: 0
```

Comment:
Should probably be 30000 ( 30s default as per RFC2205 )

Reply:
Done.